diagnosis/scripts/release.sh

```diff
@@ -4,5 +4,5 @@ VERSION=$1
 
 dos2unix *.sh
 mkdir -p _out
-tar -czf _out/diagnosis-v${VERSION}.tar.gz collectlogs.sh getkuberneteslogs.sh hosts.sh
-zip -q _out/diagnosis-v${VERSION}.zip collectlogs.sh getkuberneteslogs.sh hosts.sh
+tar -czf _out/diagnosis-v${VERSION}.tar.gz collectlogs.sh getkuberneteslogs.sh hosts.sh azs-collect-windows-logs.ps1
```

Comment:
are the ps1 files consumed?

Reply:
Yes, https://github.com/msazurestackworkloads/azurestack-gallery/blob/master/diagnosis/getkuberneteslogs.sh#L136